src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -171,36 +151,12 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
             <span className="input-error">Password is required</span>
           )}
         </div>
-        <TicketCodeField register={register} error={errors?.code} />
-        <DateOfBirthField register={register} error={errors?.date_of_birth} />
-        {IS_BURN &&
```

Comment:
This came from removing the IS_BURN reference from this file; these hardcoded code of conduct questions won't be needed anymore.